### PR TITLE
Updating current ssl cipher list

### PR DESCRIPTION
**Pull request by jmassardo**
Signed-off-by: James Massardo <jmassardo@chef.io>

### :nut_and_bolt: Description: What code changed, and why?

Update documentation to match current supported ssl ciphers

### :chains: Related Resources

The list of ciphers is in the code here: https://github.com/chef/automate/blob/master/api/config/shared/tls.go#L14-L23

### :+1: Definition of Done

Docs change only

### :athletic_shoe: How to Build and Test the Change

Docs change only

### :white_check_mark: Checklist

- [ X ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ NA] Tests added/updated?
- [ X ] Docs added/updated?
- [ X ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/docs-chef-io/content/automate/configuration.md [hunk 1]
```diff
@@ -366,7 +366,7 @@ Uncomment and change settings as needed, and then run `chef-automate config patc
 # large_client_header_buffers_size = "8k"
 # large_client_header_buffers_number = 4
 # sendfile = "on"
-# ssl_ciphers = "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:AES256-GCM-SHA384:!aNULL:!eNULL:!EXPORT"
+# ssl_ciphers = "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:!aNULL:!eNULL:!EXPORT"
```
Comment (stevendanna) [inline components/docs-chef-io/content/automate/configuration.md:369]: I think AES256-GCM-SHA384 should still be in this list.

Comment (jmassardo) [reply components/docs-chef-io/content/automate/configuration.md:369]: It's not in the `commonCiphers`. Appears to the code is appending that cipher in a section below the common list. I'll update.

Comment (jmassardo) [reply components/docs-chef-io/content/automate/configuration.md]: Done.

**Review by stevendanna — APPROVED** [review]
Thanks for fixing this up!

**Review by mjingle — APPROVED** [review]
Seconded. Thank you for updating this @jmassardo!

**Pull request merged**